### PR TITLE
Python net integration

### PR DESCRIPTION
**Pull request by medegor44**

### CODE REVIEW

File: Src/EmbeddedScripts.Python.PythonNet/PythonMultithreadingScope.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace EmbeddedScripts.Python.PythonNet
+{
+    public class PythonMultithreadingScope : IDisposable
+    {
+        private readonly IntPtr _threadState;
+
+        public PythonMultithreadingScope()
+        {
+            PythonEngine.Initialize();
```
Comment (confirmit-horizons) [inline Src/EmbeddedScripts.Python.PythonNet/PythonMultithreadingScope.cs:12]: Looks like it should be global configuration (not per-CodeRunner instances) as it's using static (global) initializers.

File: Src/EmbeddedScripts.Python.PythonNet/PythonNetRunner.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public class PythonNetRunner : ICodeRunner
+    {
+        private Container _container = new();
+        public Task RunAsync(string code)
+        {
+            using (new PythonMultithreadingScope())
+            using (Py.GIL())
+            using (var scope = Py.CreateScope())
+            {
+                foreach (var name in _container.VariableAliases)
```
Comment (confirmit-horizons) [inline Src/EmbeddedScripts.Python.PythonNet/PythonNetRunner.cs:18]: nit: can be extracted to a separate method

**Review by confirmit-horizons — CHANGES_REQUESTED** [review]

File: Src/EmbeddedScripts.Python.PythonNet/EmbeddedScripts.Python.PythonNet.csproj [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>EmbeddedScripts.Python.PythonNet</RootNamespace>
+        <LangVersion>latest</LangVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="pythonnet_netstandard_py39_win" Version="2.5.2" />
```
Comment (confirmit-horizons) [inline Src/EmbeddedScripts.Python.PythonNet/EmbeddedScripts.Python.PythonNet.csproj:10]: Probably will need a linux-x64 version of python runtime to get green tests

Comment (medegor44) [reply Src/EmbeddedScripts.Python.PythonNet/EmbeddedScripts.Python.PythonNet.csproj:10]: Situation is more complicated. I have just copied python's runtime into bin/Debug folder on Windows

**Comment by medegor44** [comment]
[Implement pythonnet runner](https://trello.com/c/ZCkLzhy3/11-implement-pythonnet-runner)

File: Tests/EmbeddedScripts.Python.PythonNet.Tests/EmbeddedScripts.Python.PythonNet.Tests.csproj [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
+        <PackageReference Include="python" Version="3.10.0-b4" />
+        <PackageReference Include="Python.Included" Version="3.7.3.13" />
```
Comment (confirmit-horizons) [inline Tests/EmbeddedScripts.Python.PythonNet.Tests/EmbeddedScripts.Python.PythonNet.Tests.csproj:12]: don't forget to remove unused refs

**Review by confirmit-horizons — APPROVED** [review]

**Pull request merged**